busybox/plan.sh

```diff
@@ -51,17 +51,36 @@ do_build() {
 do_install() {
   install -Dm755 busybox "$pkg_prefix/bin/busybox"
 
+  # Check that busybox executable is not failing
+  busybox
```

Comment:
+1

Reply:
+1 as well, but I think we should use an absolute path here so we don't accidentally test the wrong thing .  `$pkg_prefix/bin/busybox`